setup.py

```diff
@@ -169,6 +169,8 @@ def load_requirements(file_list: Optional[Union[str, List[str]]] = None) -> List
         "local-inference": all_doc_reqs,
         "paddleocr": load_requirements("requirements/extra-paddleocr.txt"),
         "embed-huggingface": load_requirements("requirements/ingest/embed-huggingface.txt"),
+        "embed-octoai": load_requirements("requirements/ingest/embed-octoai.txt"),
```

Comment:
This is good. lets change the code in octoai.py to say embed-octoai instead of embed-openai (I should have looked  deeper into this. It uses the openai code...)

Reply:
Nice one, addressed with [change extra name for octoai](https://github.com/Unstructured-IO/unstructured/pull/2693/commits/658de79d64ef55db81137cd7d5601d455ce61f07)